ingesters/ingesters.md

```diff
@@ -132,6 +133,13 @@ Most of the core ingesters support a common set of global configuration paramete
 
 The Ingest-Secret parameter specifies the token to be used for ingest authentication.  The token specified here MUST match the Ingest-Auth parameter for Gravwell indexers.
 
+### Ingest-Secret-File
+
+There may be instances where it is undesirable to load ingest secrets as cleartext strings directly from a configuration file; the `Ingest-Secret-File` parameter allows for specifying a file which contains the `Ingest-Secret`.  The `Ingest-Secret-File` should contain the full path to a file containing only the secret value, the file must be readable by the calling ingester.  If both `Ingest-Secret` and `Ingest-Secret-File` are specified, the `Ingest-Secret` value takes precedence; if the `Ingest-Secret` is populated via an environment variable, `Ingest-Secret-File` is ignored.
+
+
+See the section in [Environment Variables](/configuration/environment-variables.html) for additional methods to divorce secrets from config files.
```

Comment:
See the... what? .... section? Ingesters section? should we be providing a link to an anchor?

Reply:
I don't follow... did I screw up the link to the environment variables page?